cmd/cody-gateway/internal/limiter/limiter.go

```diff
@@ -86,6 +86,18 @@ func (l StaticLimiter) TryAcquire(ctx context.Context) (_ func(context.Context,
 		return nil, NoAccessError{}
 	}
 
+	// If the expiry on the key is greater than the intervalSeconds, update the TTL.
+	// This is here as a safeguard for the case where the TTL is wrongly set or the
+	// usage cache is not reset when the intervalSeconds config change. This only covers
+	// the case where the new intervalSeconds in shorter than the present TTL, i.e Free -> Pro.
```

Comment:
```suggestion
	// the case where the new intervalSeconds in shorter than the present TTL
```
Free / Pro are not items that this code deals with